observability/admin-metrics/serverless-admin-metrics.adoc

```diff
@@ -8,7 +8,7 @@ toc::[]
 
 Metrics enable cluster administrators to monitor how {ServerlessProductName} cluster components and workloads are performing.
 
-You can view different metrics for {ServerlessProductName} by navigating to link:https://docs.openshift.com/container-platform/latest/observability/monitoring/reviewing-monitoring-dashboards.html#reviewing-monitoring-dashboards-admin_reviewing-monitoring-dashboards[*Dashboards*] in the web console *Administrator* perspective.
+You can view different metrics for {ServerlessProductName} by navigating to link:https://docs.openshift.com/container-platform/latest/observability/monitoring/reviewing-monitoring-dashboards.html#reviewing-monitoring-dashboards-admin_reviewing-monitoring-dashboards[*Dashboards*] in the web console.
```

Comment:
I do not see a section named "Dashboards". How about point to "[About monitoring dashboards](https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html-single/monitoring/index#about-monitoring-dashboards_key-concepts)"?